### ISSUE TITLE
e2e: create an automated e2e test

### ISSUE THREAD

**Issue by crwilcox** [google/exposure-notifications-server#247]
We have done some manual runs, but it would be great to have a test that can do an automated e2e walkthrough of exposure keys getting uploaded and exposure keys getting downloaded

**Comment by radisef347** [comment]
![alt text](https://cdni.pornpics.com/1280/7/51/81056079/81056079_141_f080.jpg)

**Comment by johanbrandhorst** [comment]
@thatlkat Is this document publically available somewhere? I'd be interested in experimenting with this.

**Comment by thatlkat** [comment]
Not yet, will add readme when we have SUT in place

**Comment by nmiodice** [comment]
I'd also love to get my eyes on that document. If one were to implement such an end to end test, would a PR for it be welcome in the repo?

We are planning some work, and I want to better understand how we should prioritize efforts. Is this already being worked on in parallel? If so, when can we expect a true end to end test to be implemented -- one that is fully isolated from the deployed environment?

**Comment by johanbrandhorst** [comment]
We should reopen this, right? Is this no longer relevant?

**Comment by sethvargo** [comment]
I believe we have an e2e test now, @crwilcox @chaodaiG can you confirm?

**Comment by chaodaiG** [comment]
We have integration tests against in memory DB, testing the following scenarios:
- Publish/export keys
- Cleanup published keys
- Cleanup exported keys

https://github.com/google/exposure-notifications-server/blob/main/internal/integration/publish_test.go

So I think we have the requested "upload" part done, but no test for "download" part yet
